### DISCUSSION TITLE
Provide "open meeting" information on community page

### DISCUSSION THREAD

**Discussion by stephanbcbauer** [eclipse-tractusx/eclipse-tractusx.github.io#489] [Ideas]
## Description/Idea

More and more products are making their way into the "open source" world. Many of the products are already in tractus-x and are also already planned in GitHub.

At the moment, not all meetings are accessible to everyone because some of the necessary links are not known to everyone.

**In the future, however, we must ensure that everyone has the opportunity to participate in a daily or weekly meeting.**

For this we need the necessary information such as time and place of the meeting, dial-in link.

## Suggestion

The necessary information could be displayed directly on the product via the [community](https://eclipse-tractusx.github.io/community) page.

<img width="880" alt="image" src="https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/assets/84396022/0709ff46-c97c-49ef-808d-f03b8430ba88">

Usefull information as attributes for the product:

- _Name_: Example Daily, Weekly
- _Frequency_: If not obvious from the name
- _Date/time_:
- _Link for participation_: https://link_to_join
- _Description/Remark_: Example Agenda is in the Discussions or or or

## Additional notes

We should also make sure, that there is still the possibility to create a dynamic agenda for each meeting (for example in a discussion item) => special labels/attributes/types are needed?

This could also be used as a protokcoll?

**Comment by Siegfriedk** [comment]
Perhaps we can have a widget on docusraus to have it per product and than being able to collect it and show it in one overview page

**Reply by SebastianBezold** [reply]
As far as I can see, the info rendered in the products accordion is [stored in a json object](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/blob/main/utils/products.js) and then rendered via [`ProductOverview`](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/blob/main/src/components/ProductOverview/index.js) React component on the [community page](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/blob/main/src/pages/community.mdx#L50).
We could think about enhancing the json structure and include meeting info there. Then we can leverage it on multiple places

**Reply by stephanbcbauer** [reply]
enhancing JSON was also my idea. -> another topic, but the json is not up to date related to the the committers :)

**Comment by SebastianBezold** [comment]
Hey @stephanbcbauer, 
like the idea! I think the community page is a good starting point. We might want to restructure the page a bit. I think, the meeting info is a bit hidden, if you have to scroll over the communication rules (which are important), a "how to engage.." section and than unfolding the accordion.

My proposal would be to shorten the "how to engage.." section to a TL;DR kind of thing and move the full content to `sig-release`, where planning is done.
The meeting info could be split in my opinion. Having the "these meetings are happening" part always visible, and if relevant, links or description about how to attend in the hidden part.
What do you think?

**Reply by stephanbcbauer** [reply]
sounds very good, is there a ReactJS developer available? If not i could do it. Will try to paint some pictures before, how it could look like. i think enahncing the JSON and restructuring the page would be a good way to go.

**Comment by stephanbcbauer** [comment]
thx @Siegfriedk , @SebastianBezold

**Comment by jjeroch** [comment]
@stephanbcbauer I like the idea - voting for it
is it possible to add a automatic calendar entry "hyperlink"?

**Reply by stephanbcbauer** [reply]
@jjeroch thx . will check

**Reply by SebastianBezold** [reply]
We could add `.ics` files to the website and provide that as download in the meeting description. So should be possible, yes 😁

**Comment by stephanbcbauer** [comment]
So right now EDC and also BPDM are planning with open weekly.  And I would like to share this information with the community on the page. Even if it is right now an interim (not so nice solution) with pure MD :)

**Comment by stephanbcbauer** [comment]
There is already an issue #523 by @SebastianBezold. I think “all” requirements would be fulfilled with this issue. My feeling, the discussion can be closed.

**Reply by SebastianBezold** [reply]
Yep. Just now, I also opened a PR with the initial implementation. See #552.
We can continue the discussion on the PR, or open dedicated issues with improvements

**Reply by stephanbcbauer** [reply]
Like your PR ;) will close this discussion. THX for your input and effort. I think the new structure of the community section rocks.
